code/__HELPERS/names.dm

```diff
@@ -15,10 +15,6 @@ var/church_name = null
 
 	return name
 
-var/command_name = null
-/proc/command_name()

```

Comment:
No, don't remove this. It's there as a layer between the map datum and rest of the code, and it's for compatibility.